dazzle.yaml

```diff
@@ -72,23 +72,23 @@ combiner:
         - lang-rust
     - name: dotnet-vnc
       ref:
-      - base
+      - full
```

Comment:
Why does dotnet need 'full'?

Reply:
Good question. It used to have full as its base image. In other words, it's a breaking change to users, to change the base image for dotnet from full to base, because they have dependencies on things that were part of full even while using the prior dotnet image.

For now, we should fix the regression, and later either plan a separate effort to make slimmer images (or build them dynamically from chunks altogether).